integration_tests/chain.go

```diff
@@ -278,14 +279,14 @@ func (c *chain) sendMsgs(clientCtx client.Context, msgs ...sdk.Msg) (*sdk.TxResp
 		}
 	}
 
-	txf.WithFees("246913560testsomm")
+	txf.WithFees("246913560usomm")
 
 	txb, err := tx.BuildUnsignedTx(txf, msgs...)
 	if err != nil {
 		return nil, err
 	}
 
-	txb.SetFeeAmount(sdk.Coins{{Denom: "testsomm", Amount: sdk.NewInt(246913560)}})
+	txb.SetFeeAmount(sdk.Coins{{Denom: "usomm", Amount: sdk.NewInt(feeAmount)}})
```

Comment:
Same here re: harcoded usomm string.

Reply:
there is a strange behavior here that isn't seeing the `testDenom` const in scope even though they are part of this file is in the same package. Trying to add a local `testDenom` const (just for testing) results in an error because it's already defined elsewhere. Haven't figured it out yet.

Reply:
Bumping this to see if you were able to figure it out.

Reply:
Yes,`*_test.go` files are scoped separately from non-test files even though they are in the same package.